plugins/python-build/share/python-build/pyston-0.5.1-numpy

```diff
@@ -0,0 +1 @@
+source "${BASH_SOURCE%/*}/pyston-0.5.1"
```

Comment:
I don't think we need separate definitions of `pyston-0.5.1` and `pyston-0.5.1-numpy`. Just make `pyston-0.5.1` as "battery included" at least for now.